backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -46,6 +66,18 @@ public static final LocalDate parseUserShortDate(String d) {
     }
   }
 
+  public static final LocalDate parseUserShortDate(String input) {
```

Comment:
this is named `parseUserShortDate` but it's actually `parseUserDate` that uses the "SHORT_FORMATTER"

Reply:
Removed the duplicate method and the SHORT_FORMATTER!

Reply:
thanks for cleaning up the naming!